### PR TITLE
Update documentation around setters

### LINKED ISSUE #13: Adjust docs for clarity around setters (RFC)
## Description
The generated docs pick up `closeButtonContent(content)` as a function which takes in a content param, but there's no documentation around the fact that it is a setter. Meaning, its usage is `closeButtonContent = yourContent`, not `closeButtonContent(yourContent)`.

## Solutions?
Maybe this is as easy as outlining this in the comments. Or is there a better way in JSDoc to deal with this sort of thing?

### PR DESCRIPTION
**Pull request by andrewrubin**
Closes #13 

This is not super pretty from a documentation perspective, but as I mentioned in #13 , there doesn't seem to be an obvious way to convey setter usage in JSDoc…

### CODE REVIEW

**Review by marlonmarcello — APPROVED** [review]
@andrewrubin it makes sense to me, also the concept of getter/setter is broad enough that I think we are safe to assume most developers will know about it.

**Comment by andrewrubin** [comment]
> @andrewrubin it makes sense to me, also the concept of getter/setter is broad enough that I think we are safe to assume most developers will know about it.

I tend to agree. The reason I filed this issue in the first place though, was because I was asked why setting modal content wasn't working per the documentation. I'm okay if we don't merge this though, if you guys prefer? @marlonmarcello @liamegan

**Comment by marlonmarcello** [comment]
Sorry man, my bad english.
I meant that the updates made sense to me, as in approved and I like it.
Thanks for these! 🎉

**Pull request merged**
